ci/pyuvdata_tests.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - pytest-cases>=3.8.3
   - pytest-cov
   - pytest-xdist
-  - cython
+  - cython>=3.0
```

Comment:
This should also be updated in the readme and maybe in setup.py?

Reply:
Good call thanks for pointing those out.